src/replica/duplication/replica_duplicator.cpp

```diff
@@ -35,17 +35,25 @@
 #include "load_from_private_log.h"
 #include "replica/mutation_log.h"
 #include "replica/replica.h"
+#include "task/task_code.h"
 #include "utils/autoref_ptr.h"
 #include "utils/error_code.h"
+#include "utils/flags.h"
 #include "utils/fmt_logging.h"
 
 METRIC_DEFINE_counter(replica,
                       dup_confirmed_mutations,
                       dsn::metric_unit::kMutations,
                       "The number of confirmed mutations for dup");
 
-namespace dsn {
-namespace replication {
+DSN_DEFINE_bool(replication,
+                load_from_private_log_level_common,
```

Comment:
False `load_from_private_log_level_common` does not necessarily mean `low`: it may be `high`. I think we could use `DSN_DEFINE_string` to define the task name directly, and initialize the code object by `task_code::try_get`.